workflow/sample_files/base-bldgtype-single-family-attached.xml

```diff
@@ -122,6 +122,18 @@
               <AssemblyEffectiveRValue>23.0</AssemblyEffectiveRValue>
             </Insulation>
           </RimJoist>
+          <RimJoist>
+            <SystemIdentifier id='RimJoistOther'/>
+            <ExteriorAdjacentTo>basement - conditioned</ExteriorAdjacentTo>
+            <InteriorAdjacentTo>basement - conditioned</InteriorAdjacentTo>
+            <Area>28.0</Area>
+            <SolarAbsorptance>0.7</SolarAbsorptance>
+            <Emittance>0.92</Emittance>
+            <Insulation>
+              <SystemIdentifier id='RimJoistOtherInsulation'/>
+              <AssemblyEffectiveRValue>23.0</AssemblyEffectiveRValue>
```

Comment:
If you make the previous change correctly, this should change to R-4.0 (while the other rim joist remains R-23.0).